modules/tensor_mechanics/src/materials/abaqus/AbaqusUMATStress.C

```diff
@@ -44,6 +44,9 @@ AbaqusUMATStress::validParams()
   params.addParam<MooseEnum>("decomposition_method",
                              ComputeFiniteStrain::decompositionType(),
                              "Method to calculate the strain kinematics.");
+  params.addCoupledVar("displacements", 0.0, "displacement variables");
```

Comment:
make it either an empty vector or no default (if I recall correctly an invalid vector parameter is just an empty vector)